client/src/components/studentsComponents/tables/StudentGroupTable.js

```diff
@@ -103,9 +104,25 @@ const StudentGroupTable = () => {
     fetchData();
   }, []);
 
+  const handleCreateGroup = async () => {
+    try {
+      // Logic to add a group
```

Comment:
We have the logic already, it's similar to what we did for the professor's view here:
https://github.com/umple/AcademicTeamManagement/blob/main/client/src/components/professorComponents/tables/GroupTable.js#L275C5-L288